### PR TITLE
[MASTER] Proposal: replace custom webpack build by CRA

### PR DESCRIPTION
**Pull request by NTag**
- Use CRA with Typescript template instead of a custom webpack configuration.

It reduces the dependencies and the size of `node_modules/` and limit the maintenance.

The result is the same:
CleanShot 2020-09-22 at 11 50 34@2x

### CODE REVIEW

**Comment by NTag** [comment]
Hum, sunbeams isn't public on npm :/ .

**Comment by NTag** [comment]
So, I replaced sunbeams with the previous custom team and Text/Icon components. But I replaced react-native-vector-icons, which need react-native-web and some webpack setup by react-icomoon. It seems to work well 👌 .

File: src/common/theme.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  ],
+  ctaBlue: [
+    "#27349B",
+    "#283AC7", // primaryActionColorActive
+    "#4B5FFA", // primaryActionColor
+    "#A8BAFF", // primaryActionColorInactive
+    "#D7E0FD", // primaryActionColorDisabled
+    "#EBF0FF", // backgroundActionableActive
+    "#F6F6FE", // backgroundActionable
+    "#FCFCFE", // backgroundActionableDisabled
```
Comment (Kui-kui) [inline src/common/theme.ts:25]: I'd remove the comments. It may confuse the candidate more than anything else 😅

Comment (NTag) [reply src/common/theme.ts]: Agreed, removed 👍

**Review by Kui-kui — APPROVED** [review]

**Pull request merged**